Frontend/src/components/Sidebar/PrivateSideBar.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useEffect } from "react";
+import { ListGroup, Collapse } from "react-bootstrap";
+import { Link, useNavigate } from "react-router-dom";
+import "../../styles/NavBar.css";
+import AuthService from "../../services/auth.service";
+import UserService from "../../services/user.service";
+
+function PrivateSideBar({ menu, students, auth, handleClick }) {
```

Comment:
Good work with the render. 👍 

I was looking for `useContext()` to derive the sidebar state, but this will do just fine. :))